Fixed an issue with input files with spaces in them.

MartinBlondin opened the pull request:
Also made use of the string.format() function to make the command strings prettier

cj81499 commented:
If cary wants to go forward with python 3 over python 2, we should use [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python), rather than .format(). They're more readable, and [perform better](https://hackernoon.com/a-closer-look-at-how-python-f-strings-work-f197736b3bdb).

jontonsoup4 commented:
I didn't even know f strings were a thing! So it'd be something like:
```python
TF=TEMP_FOLDER
f'ffmpeg -framerate {frameRate} -i "{TF}/newFrame%06d.jpg" -i "{TF}/audioNew.wav" -strict -2 {OUTPUT_FILE}'
```

carykh commented:
I think I actually am using Python 3 instead of 2. I was confused because on my computer, the command is simply "python", and I also have "python3", so I assumed python = python2. But that's not true, haha

carykh commented:
So with that being said, I'll let somebody implement f strings and then I'll merge the pull request. (I guess I could do it myself, but the power of crowdsourcing is cool)

MartinBlondin commented:
I used format because it suprisingly worked even with the python 2 interpreter.

There is a bug in this implementation though, the very last file path isnt inside quotations, oops.

I have fixed this and implemented f-strings in this pull request, Although this will make the Python version requirement 3.6.

I also see someone has beat me to it: https://github.com/carykh/jumpcutter/pull/20

He has not however fixed the file path issues so perhaps this pull request could be merged after his.
